github/azure-self-hosted-runners/config.toml.tmpl

```diff
@@ -0,0 +1,121 @@
+[default]
+# This URL is used by instances to send back status messages as they install
```

Comment:
~Can we purge the comments and link to the upstream docs for it?~ I guess it is fine.